changelogs/fragments/pep8_bugfix.yml

```diff
@@ -0,0 +1,4 @@
+---
+bugfixes:
```

Comment:
these are trivial changes, would you please update the changelog type

Reply:
I have replaced by `minor_changes`